### PR TITLE
Update app and GUI

### LINKED ISSUE #171: Update GUI for user experience and align it with current product
Image

1. Currently in the GUI, the field names are not present so it may be confusing for the user for certain fields such as main number and other numbers.
2. Restrict length of emails and addresses to 200
3. Update add command example to better reflect with the product as it still has things like owe money and friends.
4. Improve error messages for labels, email, addresses.
5. Update default data to better reflect with the product.

### PR DESCRIPTION
**Pull request by Zisaac99**
Fixes #171.

Whats new,
1. Added field names to GUI.
2. Restrict length of emails and addresses to 200.
3. Update add command example to better reflect with the product as it still has things like owe money and friends.
4. Improve error messages for labels, email, addresses.
5. Update sample data to better reflect with the product.

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/ParserUtilTest.java [hunk 1]
```diff
@@ -121,6 +121,30 @@ public void parseAddress_invalidValue_throwsParseException() {
         assertThrows(ParseException.class, () -> ParserUtil.parseAddress(INVALID_ADDRESS));
     }
 
+    @Test
```
Comment (jvvvstin) [inline src/test/java/seedu/address/logic/parser/ParserUtilTest.java]: Nice that you added test cases to test the newly implemented max address length. Likewise for emails

File: src/test/java/seedu/address/logic/parser/ParserUtilTest.java [hunk 2]
```diff
@@ -121,6 +121,30 @@ public void parseAddress_invalidValue_throwsParseException() {
...
+    public void parseAddress_invalidLength_throwsParseException() {
+        String invalidAddressLength = "15, 29a Upper Paya Lebar Road, Singapore 533015 (Home 1) "
+                + "Blk 742, 04-187, Yishun Avenue 5, Singapore 760742 (Home 2) 290, 03-22, Orchard Road, "
+                + "Singapore 238801 (Home 3) Jurong West Street 2112 (Home 4)";
+
+        assertThrows(ParseException.class, () -> ParserUtil.parseAddress(invalidAddressLength));
+    }
+
+    @Test
+    public void parseAddress_validValueLengthJustNice_returnsAddress() throws Exception {
```
Comment (MooMooYonders) [inline src/test/java/seedu/address/logic/parser/ParserUtilTest.java]: Good that you tested the boundary value length
